packages/cli/src/cliEntry.js

```diff
@@ -12,7 +12,7 @@ import childProcess from 'child_process';
 import commander from 'commander';
 import path from 'path';
 
-import type {CommandT, ConfigT} from './tools/config/types.flow';
+import type {CommandT, ConfigT} from '../../../types/config';
```

Comment:
I wonder if we could somehow tell `flow` to look it up from the `root` w/o specifying the relative path?